doc/whats_new.rst

```diff
@@ -254,7 +254,7 @@ New features
    - The new module :mod:`nilearn.connectome` now has class
      :class:`nilearn.connectome.ConnectivityMeasure` can be useful for
      computing functional connectivity matrices.
-   - The function :func:`nilearn.connectome.sym_to_vec` in same module
+   - The function :func:`nilearn.connectome.sym_matrix_to_vec` in same module
```

Comment:
Now that 0.2.6 is released. Can you please update ?
ping @GaelVaroquaux @AlexandreAbraham next going to be 0.2.6.1 or 0.2.7 ?